doc/zh_CN/README.md

```diff
@@ -0,0 +1,118 @@
+# Buildbarn的Storage守护进程
+

```

Comment:
I am not able to keep this document up-to-date myself, unfortunately. This means that this document may in practice lag behind the English version of the README. Could you please add a tiny notice at the top, containing the date at which this translation was performed? That way users have a better idea of whether the documentation they are reading is up-to-date. Thanks!

> *This document was translated from the English version on 2021-01-25.*